xbmc/cores/VideoPlayer/DVDOverlayContainer.cpp

```diff
@@ -24,6 +24,8 @@ void CDVDOverlayContainer::Add(CDVDOverlay* pOverlay)
 
   CSingleLock lock(*this);
 
+  bool add = true;
```

Comment:
I would maybe call the var `addToOverlays` instead of `add`. Only because the word `add` has many meanings. But just a suggestion. Entirely up to you.

Reply:
Will do.
Maybe you can also think of a different name for this method. `Add()` implies that the method just adds the overlay into the container. However this method doesn't only add the overlay but also does some sort of preprocessing on the container itself. When trying to understand the code, the logic here got unnoticed for a while because of the name.

Reply:
ProcessAndAddOverlayIfValid()?

Reply:
That sounds right. Will adjust whenever @MediaBrasil confirms he did not spot any regression in his iso file.